aiohttp/client_exceptions.py

```diff
@@ -238,17 +239,31 @@ class InvalidURL(ClientError, ValueError):
 
     # Derive from ValueError for backward compatibility
 
-    def __init__(self, url: Any) -> None:
+    def __init__(self, url: Any, description: Optional[str] = None) -> None:
         # The type of url is not yarl.URL because the exception can be raised
         # on URL(url) call
-        super().__init__(url)
+        super_args = [url]
+        if description:
+            super_args.append(description)
+        super().__init__(*super_args)
```

Comment:
Think I'd prefer the more direct approach here, personally.
```suggestion
        if description:
            super().__init__(url, description)
        else:
            super().__init__(url)
```

Reply:
Sure, updated.